src/Microsoft.DotNet.XHarness.Common/CLI/ExitCode.cs

```diff
@@ -105,4 +105,10 @@ public enum ExitCode
     /// Hardware device is in some corrupted state, or just locked screen
     /// </summary>
     DEVICE_FAILURE = 89,
+
+    /// <summary>
+    /// This timeout occurs when the Apple app has been launched but hasn't
+    /// connected yetover TCP and --launch-timeout expires
+    /// </summary>
+    APP_LAUNCH_TIMEOUT = 90,

```

Comment:
Is this specific to Apple only? If so, shouldn't this be reflected in the enum name, or you want to have a generic category for this kind of faults even though we observe this just on Apple?

Reply:
Yeah, it is possible we will re-use this in Android in future. We only agreed to have the description reflect the scenarios each code is used in.